freud/util.pxd

```diff
@@ -19,19 +19,19 @@ ctypedef float complex fcomplex
 ctypedef double complex dcomplex
 
 ctypedef enum arr_type_t:
-    UNSIGNED_INT
     FLOAT
     DOUBLE
     COMPLEX_FLOAT
     COMPLEX_DOUBLE
+    UNSIGNED_INT
```

Comment:
Why did you reorder this? If you wanted to be alphabetical you need to change all the others too, it was virtually in reverse alphabetical order before.

Reply:
This is just the result of a conflict resolution -- I didn't do this on purpose. I don't think the order matters as long as we're consistent, which is currently true.